extensions/sockets/src/test/resources/udp-send-config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:sockets="http://www.mulesoft.org/schema/mule/sockets"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/sockets http://www.mulesoft.org/schema/mule/sockets/current/mule-sockets.xsd">
+
+    <sockets:requester-config name="udp-requester">
+        <sockets:udp-requester-connection host="localhost" port="${port_udp}">
+            <sockets:udp-requesting-socket-properties>
+                <sockets:default-udp-requesting-socket-properties broadcast="false"/>
```

Comment:
sounds like this object should actually be a parameter group of the requester connetion... This XML is horrible and will not translate well into mozart nor Studio